scripts/collect_interest.js

```diff
@@ -1,21 +1,25 @@
 const fse = require("fs-extra");
-const settings = require("./deploy-settings.json");
+const settings = require("../stakingModel/migrations/deploy-settings.json");
 const StakingContract = artifacts.require("./SimpleDAIStaking.sol");
 const GoodFundsManager = artifacts.require("./GoodFundManager.sol");
 
-module.exports = async function(deployer, network) {
-  //currently this is disabled
-  if (network) {
-    console.log("this migration is disabled");
-    return;
+const { networks } = require("../stakingModel/truffle-config.js");
+
+async function getNetworkName() {
+  const networkId = await web3.eth.net.getId();
+  for (let name in networks) {
+    if (networks[name].network_id === networkId.toString())
+      return name;
   }
+};
+
+module.exports = async function() {
```

Comment:
i'm not sure that you actually ran this scripts
because you need to accept a callback and call it when you are done, other wise the script will never finish
see my script...
also see my script how i actually export a function that wraps the main function and catches exceptions

Reply:
d866cf70b148f4de371ffd5ab20be890079ebb88